packages/@o3r/design/src/core/design-token/renderers/design-token-style.renderer.ts

```diff
@@ -28,16 +29,33 @@ export const compareVariableByName = (a: DesignTokenVariableStructure, b: Design
 
 /**
  * Sort Token variable by name using the local alphabetical order
- * @param _variableSet Complete set of the parsed Design Token
+ * @param _variables Complete set of the parsed Design Token
+ * @param options Parameters of the Design Token list transform function
  */
-export const getTokenSorterByName: TokenListTransform = (_variableSet) => (tokens) => tokens.sort((a, b) => a.getKey().localeCompare(b.getKey()));
+export const getTokenSorterByName: DesignTokenListTransform = (_variables, options) => {
+  const splitNameRegExp = /^(.*?)(\d+)$/;
```

Comment:
Isn't `^(.*)(\d+)$` enough ?
Maybe I just don't understand the `?` here.

Reply:
`*?` is for non-greedy `*`, cf https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions/Quantifiers

Reply:
ooooh I learn something new so early today ! 😄

Reply:
what about `/^([^\d]*)(\d+)$/` ?

Reply:
> what about `/^([^\d]*)(\d+)$/` ?
not matching the Token `var-1-black-100`